test_typing_inspect.py

```diff
@@ -153,10 +247,16 @@ def test_parameters(self):
         self.assertEqual(get_parameters(int), ())
         self.assertEqual(get_parameters(Generic), ())
         self.assertEqual(get_parameters(Union), ())
-        self.assertEqual(get_parameters(List[int]), ())
+        if not LEGACY_TYPING:
+            self.assertEqual(get_parameters(List[int]), ())
+        else:
+            # in 3.5.3 a list has no __args__ and instead they are used in __parameters__
+            # in 3.5.1 the behaviour is normal again.
+            pass
```

Comment:
I think the whole else part is not needed (IIUC we are not going to support 3.5.1 anyway).

Reply:
Same comment than below - I would leave it as it does not hurt :)